ClearKeep/Scenes/Call/Views/InComingVoiceCallView.swift

```diff
@@ -0,0 +1,191 @@
+//
+//  InComingVoiceCallView.swift
+//  ClearKeep
+//
+//  Created by đông on 03/04/2022.
+//
+
+import SwiftUI
+
+import SwiftUI
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/9d22ca8cb862d8c613cfefddad57a36d8fff42d1/ClearKeep/Scenes/Call/Views/InComingVoiceCallView.swift#L10"></span>Imports should be unique.<br />
<code>duplicate_imports</code> <code>InComingVoiceCallView.swift:10</code></td>
    </tr>
  </tbody>
</table>